spec/features/course/assessment/submission/submissions_spec.rb

```diff
@@ -7,6 +7,7 @@
   with_tenant(:instance) do
     let(:course) { create(:course) }
     let(:assessment) { create(:assessment, :with_all_question_types, course: course) }
+    let(:mcq_assessment) { create(:assessment, :published, :with_mcq_question, course: course) }
```

Comment:
Not possible to use the assessment in line 9?

Reply:
not all types of questions are prefillable (only MCQ and Programming). Since the addition of the testcase is basically to ensure our prefill feature works, it's important to keep it simple (only isolate it towards those questions which types are indeed prefillable).

Though, in hindsight, maybe I need to add some testcase for programming, but this require very complicated setup (need to actually attempt the answers, evaluate, then check if all public, private, and evaluation testcases pass)